### DISCUSSION TITLE
Recycling BC

### DISCUSSION THREAD

**Discussion by chanleonzen** [Nek5000/nekRS#285] [General]
Hello,

I am planning to run a simulation with the recycling BC using nekRS. I noticed that there is an example on how you would do it on nekRS (turbPipe) but it seems like it is for a structured mesh where you know the offset slabs of elements. I was wondering if it is possible to use the recycling BC based on some offset length (eg. find points method in nek5000)?
Many thanks in advance.

Kind regards,
Leon

**Comment by stgeke** [comment]
This is not supported yet.

**Reply by chanleonzen** [reply]
Thanks for clarifying!
